proto/primitives/astria/primitive/v1/types.proto

```diff
@@ -55,3 +55,10 @@ message Address {
   reserved 1;
   reserved "inner";
 }
+
+// A `TransactionId` is a unique identifier for a transaction.
+// It contains the hash of the transaction, to be included in
+// rollup deposit events for source tracking.
+message TransactionId {
+  string hash = 1;
```

Comment:
IMO prefer to call this `string inner = 1;` or even `hex inner = 1;` Saying that this is a hash (of what?) is not really important.

Please add which restrictions this field has (namely, 32 bytes lower-hex encoded).